website/docs/concepts/components/children.mdx

```diff
@@ -250,3 +250,7 @@ pub fn render_page(with_sidebar: bool) -> Html {
     }
 }
 ```
+:::note Extra Braces Required
+`sidebar = {{html_nested!{...}}}` appears to require one extra set of braces.
+This is unintuitive, and we will fix the behavior in a later release.
+:::
```

Comment:
add notes on the website for https://github.com/yewstack/yew/issues/2267

This help check the correctness of the combined website workflow too